OsuPlayer/Views/EditUserView.axaml.cs

```diff
@@ -205,10 +202,25 @@ private async void SaveChanges_OnClick(object? sender, RoutedEventArgs e)
                 return;
         }
 
-        ProfileManager.User = ViewModel.CurrentUser;
+        if (ViewModel == null)
+        {
+            ProfileManager.User = await ApiAsync.GetProfileByNameAsync(editUserModel.UserModel.Name);
 
-        if (ViewModel.IsNewProfilePictureSelected)
-            await UpdateProfilePicture();
+            if (ViewModel.IsNewProfilePictureSelected)
```

Comment:
Won't this fail if the ViewModel is checked to be null before and we check a property of it here?

Reply:
Yes this is true, I forgot to add it into my commit. I pushed it